### DISCUSSION TITLE
New name

### DISCUSSION THREAD

**Discussion by erlend-sh** [raffomania/linkblocks#40] [General]
https://lemmy.world/comment/9058142

> that name feels counterintuitive. i dont read ‘bookmarks’ from linkblock, i read ‘filter’

> *raffomania said:*
> That’s a good point - I havent thought about it that way. I’ve mostly picked that name to have something to work with, I’m open to any suggestions!

Suggestions from the community so far:
    Link
    Lynk
    Lync
    Linker
    Linkist
    LinkPage
    PageSave
    PageSaver
    PageList
    SavePage
    WebMarks
    Webory (WebMemory)
    Webist
    Netmarks

---

I also found this line from the [introduction post](https://www.rafa.ee/articles/introducing-linkblocks-federated-bookmark-manager/) inspiring:

> Bookmarking and linking on linkblocks is like publishing your own small website, saying “This is the good stuff”.

'Good Stuff' is quite whimsical and pleasant.

**Comment by raffomania** [comment]
I like "Good Stuff"! 

Other ideas:

- LinkLink
- WebWeb
- LinkLib
- LinkPub
- LinkHub
- LinkNet

**Reply by erlend-sh** [reply]
LinkStuff

**Comment by erlend-sh** [comment]
I just put ‘link’ in a thesaurus and one of the words it returned was: tie, as in *tying* stuff together. Very short, and remarkably underused for software if you search for it among github repos.

A [rolled up tie](https://www.tiesplanet.com/images/wine-red-rib-patterned-mens-tie-p9262-21710_zoom.jpg) also looks like a bookmark, so that could make for a pretty great logo 😁 

Alternatively a bowtie 🎀 looks a lot like a URL.

**Reply by raffomania** [reply]
This is a really great idea. It's a super flexible verb for publicity work and educational material as well :)

**Reply by raffomania** [reply]
- "webtie"
- "bundletie"
- "tie together" (nod towards collaboration)

**Comment by raffomania** [comment]
Despite (or because of?) the overused term "digital gardening", I keep coming back to ideas like "webgarden" or "gardening links".

None of the ideas I've thought of have that little something to make them memorable, though...

**Reply by erlend-sh** [reply]
how about..

*Trees* 🌳 🌲 🌴 

?

Every tree represents a high-centrality node in a fully distributed network, interlinked by an underground [wood-wide web](https://en.wikipedia.org/wiki/Mycorrhizal_network).

Other names in that vein (from that Wikipedia article):
- Symbiosis, Symbi, Symbs.. links have symbiotic relationships with each other.
- Mutualistic, Mutual, Muti..
- Fungi

> I keep coming back to ideas like "webgarden" or "gardening links".

Yeah, or “links cultivation”. My links repository won’t be be carefully *gardened*. It’s more like I’m just leaving the grass of my lawn to grow freely.

**Reply by raffomania** [reply]
Yes! *link forestry*, what a nice job that would be.
You could also think of backlinks as the leaves of a tree, where new information grows. normal links then are the roots, where you can trace the origins of knowledge that is transported towards the leaves.

More name ideas: "linktrees", "linkforest", "forest of the web", or maybe just "trees". "wood-wide web" is fantastic, it would also make a catchy term for blog posts and toots :)

This also reminds me of [mycorrhiza wiki](https://mycorrhiza.wiki/) which uses fungi-related terms for their functionality.

**Reply by erlend-sh** [reply]
I really love [forestry](https://en.wikipedia.org/wiki/Forestry) here.

> Forestry is the science and craft of creating, managing, planting, using, conserving and repairing forests and woodlands for associated resources for human and environmental benefits.

It aligns beautifully with this recent call to action: [We Need To Rewild The Internet](https://www.noemamag.com/we-need-to-rewild-the-internet/).

A social link directory revolves around a practice of collection & cultivation. In that regard, *forestry* is more evocative than *trees*, since *forestry* is both a science and a craft. Links forestry is the macro to digital gardening's micro.

Calling the app Forestry and making its tag line something to do with "link forestry" is perfect in my opinion. You'd also get to affectionately call participants in the network "foresters". You'll have a hard time finding a short domain name for such a common word, but it may not matter much since the success of this app will be measured in the proliferation of its distributed links-network, not a central site.

**Reply by raffomania** [reply]
It looks like the biggest product using the "forestry" name has [been sunset in 2023](https://tina.io/forestry/), this might be the opportunity to swoop in :)

**Reply by raffomania** [reply]
Domain ideas that are available:

- forestry.dev
- forestry.rs
- forestry.link
- forestry.page
- forestry.directory
- forestry.place
- forestry.website
- forestry.technology
- forestry.systems
- forestry.town
- forestry.garden

I would have loved forestry.network, but that one's taken.

**Reply by erlend-sh** [reply]
My favorites from there are .page and .website

`forestry.page` feels right. It’s a place for digital forestry.

**Comment by erlend-sh** [comment]
I'm quite sold on *forestry*, but the imagery of a large landscape did put me on another interesting tangent that I may as well note down here: Maps.

Just like maps, links merely show you where a thing is; is is not the thing itself. [The Map Is Not the Territory](https://fs.blog/map-and-territory/).

As such, some possible names to play with include: Almanac, Atlas, Lexicon, Codex...

**Reply by raffomania** [reply]
Some ideas for this one:

- webmap / mapweb (webmap.network, webmap.space and webmap.link are free)
- webatlas / atlasweb
- netmap
- linkmap
- webtrail

**Reply by erlend-sh** [reply]
Hmmm, `webmap.space` is *really good* 😁 

It’s like webmap is the macro version of a site map.

**Reply by raffomania** [reply]
Ohh yes. Also it's similar to the concept of a mind map which is pretty much how linkblocks' lists are designed to work :)

I might prefer "webmap" over "forestry" because the word rolls off the tongue, it's less complicated to pronounce. Also, it's a bit less generic.

Other free domains: webmap.place, webmap.site, webmaps.dev

**Comment by mkitzmann** [comment]
Hi everyone,
Just to add my thoughts to this discussion. I felt that the new name could show the most unique feature about this project: Federation. So I came up with these alternatives:

LinkVerse
FediLinks
Fedinks ;)
PinVerse (altough that sounds a little like perverse)
LinkShare

Also I quite liked:
Nexus
(Wikipedia: Nexus is a Latin word for connection, usually where multiple elements meet.)

**Reply by raffomania** [reply]
Thanks for these suggestions, they are great!

Seems like [FediLinks is already taken](https://fedilinks.org/). There are [two](https://linkverse.io/about/) high-ranking [search results](https://linkverseai.com/) for LinkVerse, and LinkShare is a pretty popular advertising product by Rakuten :/

Tbh, I was ready to go with webmap, but then noticed that it might be confused with embedded widgets from google maps, mapbox, etc.

**Comment by raffomania** [comment]
Thanks everyone for the great suggestions!

linkblocks is now named **ties**.

**Reply by erlend-sh** [reply]
*ties* plural is so good!
